source/jsonnet/england-wales/individual/blocks/qualifications/degree.jsonnet

```diff
@@ -9,22 +9,34 @@ local proxyTitle = {
   ],
 };
 
-local englandGuidanceTitle = 'Include equivalent qualifications achieved anywhere outside England and Wales';
-local walesGuidanceTitle = 'Include equivalent qualifications achieved anywhere outside Wales and England';
+local englandQuestionDescription = 'This could be equivalent qualifications achieved anywhere outside England and Wales';
+local walesQuestionDescription = 'This could be equivalent qualifications achieved anywhere outside Wales and England';
+
+local englandQuestionGuidanceList = [
+  'This is any higher education qualification, achieved in the UK or another country, at level 4 or above.',
+  'Students are usually aged 18 years or over when they begin these qualifications through a college or university.',
+  'Examples include level 4 and 5 qualifications, bachelor’s degrees with or without honours, master’s degrees, PhD or other doctorates and professional qualifications, such as PGCE or chartership.',
```

Comment:
These are displayed as bullet point, I think these are just meant to be paragraphs.

Reply:
yes, that's right